src/app/components/itinerary-list/itinerary-list.component.ts

```diff
@@ -0,0 +1,48 @@
+import {Component, OnInit} from '@angular/core';
+import {ItenaryService} from "../../services/itinerary.service";
+import {Trip} from "../../models/trip";
+import {Itinerary} from "../../models/itinenary";
+import {ActivatedRoute, Router} from "@angular/router";
+import {TripService} from "../../services/trip.service";
+
+@Component({
+  selector: 'app-itinerary-list',
+  templateUrl: './itinerary-list.component.html',
+  styleUrls: ['./itinerary-list.component.scss']
+})
+export class ItineraryListComponent implements OnInit{
+  itenaries:Itinerary[]=[];
```

Comment:
I told the class a million times that a trip has 1 itinerary. An itinerary is a list of activities for a trip. Sigh. Just leave it. It's too late now.